src/View/layout.html.twig

```diff
@@ -18,7 +18,7 @@
 		<nav class="navbar navbar-expand-lg">
 			<div class="container-fluid">
 				<a class="navbar-brand" href="#">
-					<img src="assets/images/logo.png" class="img-fluid logo" alt="Le plateau" >
+					<img src="/assets/images/logo.png" class="img-fluid "  alt="Le plateau" width="140" height="140">
```

Comment:
The HTML attributes width and height are forbidden, you should do the same thing in the css